lib/math/math.go

```diff
@@ -0,0 +1,198 @@
+// Copyright 2021 The Bazel Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package math provides basic constants and mathematical functions.
+package math // import "go.starlark.net/lib/math"
+
+import (
+	"math"
+
+	"go.starlark.net/starlark"
+	"go.starlark.net/starlarkstruct"
+)
+
+const (
+	tau    = math.Pi * 2
+	oneRad = tau / 360
+)
+
+var (
+	toDeg = func(x float64) float64 { return x / oneRad }
+	toRad = func(x float64) float64 { return x * oneRad }
+)
+
+// Module math is a Starlark module of math-related functions and constants.
+var Module = &starlarkstruct.Module{
+	Name: "math",
+	Members: starlark.StringDict{
+		"abs":   starlark.NewBuiltin("abs", abs),
```

Comment:
We can simplify this file substantially by defining a helper function that combines the call to NewBuiltin and oneArgFunc, like so:

```
Members: starlark.StringDict{
		"abs":   newUnaryBuiltin("abs", math.Abs),
                 ...
}

// newUnaryBuiltin wraps a unary floating-point Go function
// as a Starlark built-in that accepts int or float arguments.
func newUnaryBuiltin(name string, fn func(float64) float64) *starlark.Builtin {
    return starlark.NewBuiltin(name, func(thread *starlark.Thread, b *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
        ...body of oneArgFunc...
    })
}
```

(Ditty for newBinaryFunction.)

This makes the table do more more, and avoids the need for lots of different wrappers, and it also allows us to consolidate our documentation at Module, instead of smearing it across all the unexported wrappers.